site/en/docs/devtools/evaluate-performance/reference/index.md

```diff
@@ -434,7 +434,18 @@ Hover over a frame to view a tooltip with more information about it.
 
 {% Img src="image/admin/MX9kRvaRf3DqfkZAgoTq.png", alt="Hovering over a frame", width="800", height="542" %}
 
-**Figure 21**. Hovering over a frame
+**Figure 21.a**. Hovering over a frame
+
+The **Frames** section can show four types of frames:
+
+1. Idle frame (white). No changes.
+1. Frame (green). Rendered as expected and in time.
+1. Partially presented frame (yellow with a sparse wide dash-line pattern). Chrome did its best to render at least some visual updates in time. For example, in case the work of the main thread of the renderer process (canvas animation) is late but the compositor thread (scrolling) is in time.
```

Comment:
```suggestion
1. **Partially presented frame (yellow with a sparse wide dash-line pattern)**. Chrome did its best to render at least some visual updates in time. For example, in case the work of the main thread of the renderer process (canvas animation) is late but the compositor thread (scrolling) is in time.
```